.github/workflows/rspec.yml

```diff
@@ -67,5 +78,11 @@ jobs:
           PGHOST: localhost
           PGUSER: postgres
           RAILS_ENV: test
+          KNAPSACK_PRO_TEST_SUITE_TOKEN_RSPEC: 0038c1f0bebcc49ae3d86e30ef9fe741
```

Comment:
@edwinthinks I can update this whenever there is a more official token to use! This is just from a test project I created with Knapsack. They suggest adding it as a project secret as well: `KNAPSACK_PRO_TEST_SUITE_TOKEN_RSPEC: ${{ secrets.KNAPSACK_PRO_TEST_SUITE_TOKEN_RSPEC }}`

So someone with access to the project settings would need to help with that!